terraso_backend/apps/core/gis/parsers.py

```diff
@@ -30,6 +30,15 @@ def is_shape_file_extension(file):
     return file.name.endswith(".zip")
 
 
+def is_shape_file_zip(file):
+    with zipfile.ZipFile(file, "r") as zip_file:
+        shp_filenames = [name for name in zip_file.namelist() if name.endswith(".shp")]
```

Comment:
question: does it make sense to store `zip_file.namelist()` in a variable and reuse that?